pkgs/top-level/python-packages.nix

```
@@ -5744,6 +5744,8 @@ in {
 
   python-slugify = callPackage ../development/python-modules/python-slugify { };
 
+  python-snap7 = callPackage ../development/python-modules/python-snap7 { inherit (pkgs) snap7; };
```

preferred formatting
```suggestion
  python-snap7 = callPackage ../development/python-modules/python-snap7 {
    inherit (pkgs) snap7; 
  };
```